### ISSUE TITLE
Add support for Gitlab Scanning

### ISSUE THREAD

**Issue by brianwyka** [Optum/sourcehawk#35]
Add support for scanning remote Gitlab repositories

**Comment by brianwyka** [comment]
@ctoestreich, think you can help with this after #68 goes in?

**Comment by ctoestreich** [comment]
Yeah sure.  I have a few more types of scans I wanted to add for yml/json as well
